### ISSUE TITLE
Add "generate autoloads" command to Cask

### ISSUE THREAD

**Issue by darkfeline** [cask/cask#394]
Cask already has commands for generating the pkg file and byte compiling files.  With a "generate autoloads" command, it would be possible to develop a package directly out of `.emacs.d/elpa` (useful for personal packages), using Cask to regenerate all of the packaging related files.

It would also be useful to add a "live dev build" convenience command to do all three of these steps.

**Comment by vermiculus** [comment]
I agree this would be very useful -- perhaps as an option to the `build` command.

**Comment by Fuco1** [comment]
Note: also remember to generate autoloads for linked packages, though I'm not sure where or how that should look. (cf. https://github.com/cask/cask/issues/205#issuecomment-39870979)
